packages/conf-m4/conf-m4.1/opam

```diff
@@ -4,10 +4,10 @@ homepage: "http://www.gnu.org/software/m4/m4.html"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 authors: "GNU Project"
 license: "GPL-3.0-only"
-build: [
-  ["sh" "-exc" "echo | m4"] {os != "macos"}
-  ["sh" "-exc" "echo | gm4"] {os = "macos"}
-] 
+build: [["sh" "-exc" "echo | m4"]]
+install: [
+  ["sh" "-c" "ln -s \"$(brew --prefix m4)/bin/m4\" \"%{bin}%/m4\""] {os-family = "homebrew"}
```

Comment:
Nice. This looks like a very nice approach afaict.

Reply:
Yes, I like it.  I've since updated it to use `gm4` as it occurred to me that we don't need the version from `brew`.